.github/workflows/devbuilds.yml

```diff
@@ -188,8 +188,7 @@ jobs:
         cd applications\mne_scan\plugins\brainflowboard\brainflow
         mkdir build
         cd build
-        set "installed_path=%cd%\..\installed"
-        cmake -G "Visual Studio 15 2017" -A x64 -DMSVC_RUNTIME=dynamic -DCMAKE_SYSTEM_VERSION=8.1 -DCMAKE_INSTALL_PREFIX=%installed_path% ..
+        cmake -G "Visual Studio 15 2017" -A x64 -DMSVC_RUNTIME=dynamic -DCMAKE_SYSTEM_VERSION=8.1 -DCMAKE_INSTALL_PREFIX="%GITHUB_WORKSPACE%\applications\mne_scan\plugins\brainflowboard\brainflow\installed" ..
```

Comment:
You might have to use `-DCMAKE_INSTALL_PREFIX="%GITHUB_WORKSPACE%\mne-cpp\applications\mne_scan\plugins\brainflowboard\brainflow\installed"` instead

Reply:
@LorenzE  I think its not an issue, from logs:
```
-- Installing: D:/a/mne-cpp/mne-cpp/applications/mne_scan/plugins/brainflowboard/brainflow/build/%GITHUB_WORKSPACE%/applications/mne_scan/plugins/brainflowboard/brainflow/installed/inc/data_filter.h
  -- Installing: D:/a/mne-cpp/mne-cpp/applications/mne_scan/plugins/brainflowboard/brainflow/build/%GITHUB_WORKSPACE%/applications/mne_scan/plugins/brainflowboard/brainflow/installed/inc/board_shim.h
  -- Installing: D:/a/mne-cpp/mne-cpp/applications/mne_scan/plugins/brainflowboard/brainflow/build/%GITHUB_WORKSPACE%/applications/mne_scan/plugins/brainflowboard/brainflow/installed/inc/brainflow_exception.h
  -- Installing: D:/a/mne-cpp/mne-cpp/applications/mne_scan/plugins/brainflowboard/brainflow/build/%GITHUB_WORKSPACE%/applications/mne_scan/plugins/brainflowboard/brainflow/installed/lib/brainflow_boards.json
```
I dont get why there is %GITHUB_WORKSPACE%

Reply:
as for me it looks like env variable is not recognized, path is completely wrong there should not be 
```
/build/%GITHUB_WORKSPACE%/
``` 
in the middle

Reply:
Maybe default shell for windows is powershell and env variable should be $env: instead % %, will try